dev_tools/scripts/update_ginkgo_header.sh

```diff
@@ -81,10 +89,10 @@ while IFS='' read -r line; do
             if [ "${READING_FIRST_LINE}" != true ] && \
                [ "${CURRENT_FOLDER}" != "${PREVIOUS_FOLDER}" ]
             then
-                echo "" >> "${GINKGO_HEADER_TMP}"
+                echo -e "${END}" >> "${GINKGO_HEADER_TMP}"
             fi
             PREVIOUS_FOLDER="${CURRENT_FOLDER}"
-            echo "#include <${file}>" >> "${GINKGO_HEADER_TMP}"
+            echo -e "#include <${file}>${END}" >> "${GINKGO_HEADER_TMP}"
```

Comment:
I am not sure if windows will put in `/` or `\` for a folder structure, but if it is `\`, you do not want it to be interpreted!
If you want to add `\r` before the end of the line, I would suggest to go with `echo -n` (which omits printing a newline), and manually do the newline in a separate `echo -e ${END}`.